inc/commonitiltask.class.php

```diff
@@ -1531,10 +1531,8 @@ function tasktemplate_update(value) {
                                   'url'       => $CFG_GLPI["root_doc"]."/ajax/planningcheck.php"];
       User::dropdown($params);
 
-      echo " <a href='#' onClick=\"".Html::jsGetElementbyID('planningcheck'.$rand).".dialog('open'); return false;\">";
-      echo "&nbsp;<img src='".$CFG_GLPI["root_doc"]."/pics/reservation-3.png'
-             title=\"".__s('Availability')."\" alt=\"".__s('Availability')."\"
-             class='calendrier'>";
+      echo " <a href='#' title=\"".__s('Availability')."\" onClick=\"".Html::jsGetElementbyID('planningcheck'.$rand).".dialog('open'); return false;\">";
+      echo "&nbsp;<i class='far fa-calendar-alt'></i>";
```

Comment:
For accessibility, add `"<span class='sr-only'>".__('Availability')."</span>"` (will replace the `img@alt`.

Reply:
Thanks, done